models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php

```diff
@@ -74,20 +69,9 @@ public function setClasses($classes)
      */
     public function getLazyLoading()
     {
-        return $this->lazyLoading;
+        return true;
     }
 
-    /**
-     * @param bool $lazyLoading
-     *
-     * @return $this
-     */
-    public function setLazyLoading($lazyLoading)
```

Comment:
still need this? https://github.com/pimcore/pimcore/blob/fb48e8b95b53fa0cd8efa33de36bea01705210c5/models/DataObject/ClassDefinition/Data/ReverseManyToManyObjectRelation.php#L79

Reply:
No, didn't have any effect anyway. 👍